twarep/electrifly
# simulation table in UI

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
Edited app.py and simulation to make the UI appear

## How I Did it?
Edited app.py and simulation to make the UI appear

## How to Test?
Run app.py and open the simulation tab, verify that the forecasted table appears for the zones. 
![image](https://github.com/twarep/electrifly/assets/64234627/a90ec328-640f-4bbc-a72d-a4c140777203)


### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
